app/src/main/res/layout/dialog_fragment_parking_size.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/padding_dialog_fragment_xsmall">
+
+    <TextView
+        android:id="@+id/text_dialog_fragment_parking_size"
+        style="@style/TextStyleAllAppTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_all_general_normal"
+        android:text="@string/title_all_parking_size"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/number_input_dialog_fragment_parking_size"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_all_general_normal"
+        android:autofillHints="no"
+        android:hint="@string/input_dialog_parking_space_hint_insert_number"
+        android:inputType="number"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
```

Comment:
if you have width match parent horizontal constraints are useless

Reply:
Nice tip

Reply:
But I think I need at least one for this occasion

Reply:
solved